src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -866,13 +866,27 @@ default String lootItemAllowlist() {
         name = "Item Denylist",
         description = "Never fire notifications for these items, despite value settings.<br/>" +
             "Place one item name per line (case-insensitive; asterisks are wildcards)",
-        position = 38,
+        position = 37,
```

Comment:
Let's double check that allow list is still above deny list with this change

Reply:
confirmed
![image](https://github.com/pajlads/DinkPlugin/assets/8106344/015a7f86-d28d-49d6-b71e-e0bc74954d91)